docs/articles/documentation/using-testcafe/common-concepts/using-browsers.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+title: Using Browsers
+permalink: /documentation/using-testcafe/common-concepts/using-browsers.html
+---
+# Using Browsers
+
+TestCafe allows you to easily set up testing in different web browsers: a number of [supported browsers](#supported-browsers), or any required browser accessed through [browser provider plugins](#browser-provider-plugins).
+You just need to specify desired browsers in one of two ways:
+
+* through the command line (see [Browser List](../command-line-interface.md#browser-list))
+* programmatically (by using the [runner.browsers](../programming-interface/runner.md#browsers) method)
+
+## Supported Browsers
+
+TestCafe directly supports testing in the following browsers.
+
+Browser           |Version | Short Browser Name
+----------------- | ------ | -------------------
+Chromium          | 22+    | `chromium`
+Google Chrome     | 22+    | `chrome`
+Internet Explorer | 9+     | `ie`
+Microsoft Edge    |        | `edge`
+Mozilla Firefox   | 17+    | `firefox`
+Opera             | 11+    | `opera`
+Safari            | 6+     | `safari`
+
+You can specify the browsers during testing by using paths or short browser names (see the table above).
+
+To obtain the list of browsers installed on the current machine, run TestCafe from the command line with the [-b option](../command-line-interface.md#-b---list-browsers).
+
+## Browser Provider Plugins
+
+You can create browser provider plugins that will enable you to use any browser unsupported by default.
```

Comment:
`unsupported by default` - not cool.

We have a word `directly` now. So, maybe

```
If you need to use a browser besides those supported directly, create a browser provider plugin.
```